Task1/Project/Shapes/Models/Classes/Circle.cs

```diff
@@ -74,16 +68,30 @@ public Point Center
             }
         }
         // METHODS
-        public override void ReadFromFile(StreamReader readStream)
+        /// <summary>
+        /// Reads some information about circle from file.
+        /// </summary>
+        /// <param name="readStream">
+        /// Stream for easy consistent access to file and is only for reading from it.
+        /// </param>        
+        public override void ReadFromFile(System.IO.StreamReader readStream)
         {
-            throw new NotImplementedException();
+            string[] data = readStream.ReadLine().Split(' ');
+            center.X = double.Parse(data[0]);
+            center.Y = double.Parse(data[1]);
+            radius = double.Parse(data[2]);        
         }
-
-        public override void WtiteToFile(StreamWriter writeStream)
+        /// <summary>
+        /// Writes some information about circle to file.
+        /// </summary>
+        /// <param name="writeStream">
+        /// Stream for easy consistent access to file and is only for writing to it.
+        /// </param>
+        public override void WriteToFile(System.IO.StreamWriter writeStream)
         {
-            throw new NotImplementedException();
+            //c - means it is data for circle.

```

Comment:
use "Circle" not "c", you will probably want to add some more classes, thay also could start with "c". it could became hard to distinguish them

Reply:
Ok.

Reply:
I have just done it.